src/Components/PaperEditor/ProgrammeScriptContainer/index.js

```diff
@@ -35,6 +41,7 @@ const ProgrammeScriptContainer = (props) => {
   const [ elements, setElements ] = useState();
   const [ title, setTitle ] = useState('');
   const [ resetPreview, setResetPreview ] = useState(false);
+  const [ timeOnClick, setTimeOnClick ] = useState();
```

Comment:
I would call it just `currentTime` cause that's what you're assigning it to :)
```suggestion
  const [ currentTime, setCurrentTime ] = useState();
```